test/download_tests.cpp

```diff
@@ -29,6 +29,54 @@ TEST(DownloadTests, DownloadGzip) {
     EXPECT_EQ(cpr::ErrorCode::OK, response.error.code);
 }
 
+TEST(DownloadTests, RangeTestWholeFile) {
+#if 0
+    const int64_t download_size = 277186;
+    cpr::SslOptions sslOpts = cpr::Ssl(cpr::ssl::VerifyPeer{false}, cpr::ssl::VerifyHost{false}, cpr::ssl::VerifyStatus{false});
+    cpr::Url url{"https://curl.haxx.se/docs/manpage.html"};
+    cpr::Session session;
+    session.SetUrl(url);
+    session.SetRange(cpr::Range{0, -1});
+    session.SetSslOptions(sslOpts);
+    cpr::Response response = session.Download(cpr::WriteCallback{write_data, 0});
+    EXPECT_EQ(200, response.status_code);
+    EXPECT_EQ(cpr::ErrorCode::OK, response.error.code);
+    EXPECT_EQ(download_size, response.downloaded_bytes);
+#endif
+}
+
+TEST(DownloadTests, RangeTestLowerLimit) {
+#if 0
```

Comment:
Tests look good, but I'm not a fan of tests that require internet here. Could you have a look at implementing rages support for our [httpServer](https://github.com/libcpr/cpr/blob/master/test/httpServer.hpp) we use for testing?